### PR TITLE
Update Step 3 of creating Tests

### PR DESCRIPTION
**Pull request by TheBrunoLopes**
Fixing the following issues with the current step 3 of getting started:
- Remove the Testkube dashboard command which is not needed anymore on cloud.
- Change the Postman example to a K6 one to better highlight the value and for the users to get started faster.
- Change the examples of how users can run tests. 

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by vLia — APPROVED** [review]

File: docs/docs/articles/step3-creating-first-test.md [hunk 1]
```diff
@@ -1,161 +1,73 @@
...
-}
-```
-
-And create the test by running the command:
+Tests in Testkube are stored as a Custom Resource in Kubernetes and live inside your cluster.
 
-```sh
-testkube create test --file postman-collection.json --type postman/collection --name my-first-test
-```
+You can create your tests directly on the UI, using the CLI or deploy them as a Custom Resource.
```
Comment (jfermi) [inline docs/docs/articles/step3-creating-first-test.md:7]:
```suggestion
You can create your tests directly in the UI, using the CLI or deploy them as a Custom Resource.
```

File: docs/docs/articles/step3-creating-first-test.md [hunk 2]
```diff
@@ -1,161 +1,73 @@
...
+You can upload your test files to Testkube or you can provide your git credentials so that testkube can fetch them automatically from your Git Repo every time there's a new test execution.
 
-:::note
-This example is testing that the website https://kubeshop.io is returning a `200` status code. It demostrates a way of creating a simple Postman test. In practice, you would test an internal Kubernetes service.
-:::
+This section provides an example of creating a _K6_ test. Nevertheless, Testkube supports a long [list of testing tools](../category/test-types).
```
Comment (jfermi) [inline docs/docs/articles/step3-creating-first-test.md:10]:
```suggestion
This section provides an example of creating a _K6_ test. Testkube supports a long [list of testing tools](../category/test-types).
```

File: docs/docs/articles/step3-creating-first-test.md [hunk 3]
```diff
@@ -1,161 +1,73 @@
...
 
-## Starting a New Test Execution
+## Creating a K6 Test
+Now that you have your Testkube Environment up an running, the quickest way to add a new test is by clicking "Add New Test" on the Dashboard and select your test type:
```
Comment (jfermi) [inline docs/docs/articles/step3-creating-first-test.md:13]:
```suggestion
Now that you have your Testkube Environment up and running, the quickest way to add a new test is by clicking "Add New Test" on the Dashboard and select your test type:
```

File: docs/docs/articles/step3-creating-first-test.md [hunk 4]
```diff
@@ -1,161 +1,73 @@
...
+<img width="1896" alt="image" src="https://github.com/kubeshop/testkube/assets/13501228/683eae92-ef74-49c8-9db9-90da76fc17fc">
 
-After our test is defined as a Custom Resource, we can run it:
-
-```sh
-testkube run test my-first-test
-```
+We created the following Test example which verifies the status code of an https endpoint.
```
Comment (jfermi) [inline docs/docs/articles/step3-creating-first-test.md:16]:
```suggestion
We created the following Test example which verifies the status code of an HTTPS endpoint.
```

File: docs/docs/articles/step3-creating-first-test.md [hunk 5]
```diff
@@ -1,161 +1,73 @@
...
+ const res = http.get('https://httpbin.test.k6.io/');
+ check(res, { 'Check if status code is 200': (r) => { 
+    console.log(statusCode, "Passing? ", 200 == statusCode);
+    return r.status == statusCode }
+});
+}
 ```
 
-## Getting the Result of a Test Execution
+Testkube can import any test files from git, from your computer or by copy & pasting a string.
```
Comment (jfermi) [inline docs/docs/articles/step3-creating-first-test.md:39]:
```suggestion
Testkube can import any test files from Git, from your computer or by copy and pasting a string.
```

File: docs/docs/articles/step3-creating-first-test.md [hunk 6]
```diff
@@ -1,161 +1,73 @@
...
-testkube get executions
-```
 
-```sh title=Expected output:"
-  ID                       | NAME              | TEST NAME       | TYPE               | STATUS | LABELS
----------------------------+-------------------+-----------------+--------------------+--------+---------
-  63f4d0910ca9ed26798741ca | my-first-test-1   | my-first-test   | postman/collection | passed |
-                           |                   |                 |                    |        |
-```
+And Voila! You can now run the test!
```
Comment (jfermi) [inline docs/docs/articles/step3-creating-first-test.md:46]:
```suggestion
Voila! You can now run the test!
```

File: docs/docs/articles/step3-creating-first-test.md [hunk 7]
```diff
@@ -1,161 +1,73 @@
...
+<img width="1896" alt="image" src="https://github.com/kubeshop/testkube/assets/13501228/e2d46e4f-641b-49b9-8a1f-f3b3100c4ad0">
 
-Copy the ID of the test and see the full details of the execution by running:
 
-```sh
-testkube get execution 63f4d0910ca9ed26798741ca
-```
+## There's Different Mechanisms to Run the Tests
```
Comment (jfermi) [inline docs/docs/articles/step3-creating-first-test.md:50]:
```suggestion
## Different Mechanisms to Run Tests
```

File: docs/docs/articles/step3-creating-first-test.md [hunk 8]
```diff
@@ -1,161 +1,73 @@
...
+#### Dashboard
+You can trigger their execution manually on the Dashboard
```
Comment (jfermi) [inline docs/docs/articles/step3-creating-first-test.md:52]:
```suggestion
Trigger test execution manually on the Dashboard:
```

File: docs/docs/articles/step3-creating-first-test.md [hunk 9]
```diff
@@ -1,161 +1,73 @@
...
+**Changing the Output Format**
 For lists and details, you can use different output formats via the `--output` flag. The following formats are currently supported:
 
 - `RAW` - Raw output from the given executor (e.g., for Postman collection, it's terminal text with colors and tables).
 - `JSON` - Test run data are encoded in JSON.
-- `GO` - For go-template formatting (like in Docker and Kubernetes), you'll need to add the `--go-template` flag with a custom format. The default is `{{ . | printf("%+v") }}`. This will help you check available fields.
\ No newline at end of file
+- `GO` - For go-template formatting (like in Docker and Kubernetes), you'll need to add the `--go-template` flag with a custom format. The default is `{{ . | printf("%+v") }}`. This will help you check available fields.
+
+#### Other means of triggering tests
```
Comment (jfermi) [inline docs/docs/articles/step3-creating-first-test.md:67]:
```suggestion
#### Other Means of Triggering Tests
```

**Review by jfermi — COMMENTED** [review]
Some editing suggestions.

**Comment by TheBrunoLopes** [comment]
@jfermi accepted ✅ 
Should I merge ?

**Pull request merged**